### ISSUE TITLE
Youtube does not process exported videos from Lossless Cut, even Windows Video Editor gives error when importing Lossless cut exported videos.

### ISSUE THREAD

**Issue by Dagg3rFall** [mifi/lossless-cut#1156]
**Bug Description:**

I imported 1 mp3 and 1 coverart exported as mp4 , uploaded on youtube, youtube uploads successfully but unable to process video.
I imported mp4 file and 1 coverart exported as mp4, uploaded on youtube, youtube uploads successfully but unable to process video.
I imported exported mp4 video from Lossless Cut to Windows Video Editor, right away I get an error.
I imported 1 mp4 video without coverart, split the video with 1 coverart. Tried uploading the video to youtube same error.

In conclusion I get a processing error for every exported video when uploaded to youtube.

**Desktop:**
 - OS: Windows 10

**Additional context:**
I have tried exporting to .mp4 and .mov and tried uploading to youtube.

![isssue with software2](https://user-images.githubusercontent.com/105827772/169711455-c8dfbc4d-8299-4b96-8963-d7df0251764a.png)
![isssue with software](https://user-images.githubusercontent.com/105827772/169711457-25c040b8-dccc-43d3-87f7-78bf25369575.png)

**Comment by mifi** [comment]
> I imported 1 mp3 and 1 coverart exported as mp4 , uploaded on youtube, youtube uploads successfully but unable to process video.

I think mp3 is not supported in mp4, so that explains why

> I imported mp4 file and 1 coverart exported as mp4, uploaded on youtube, youtube uploads successfully but unable to process video.

does it work without cover art? I assume you added it as an "attachment" as described here? https://www.youtube.com/watch?v=4pYJ93cn80E

could you try a different JPEG file and see if that works?

> I imported 1 mp4 video without coverart, split the video with 1 coverart. Tried uploading the video to youtube same error.

not sure what "split with 1 coverart means" but maybe you can share a "report" from the help menu after loading the file you're trying to upload to youtube?

**Comment by hejhub** [comment]
I'm facing the same issue, 
I added a 15 sec .mp4 video and 1 min .wav audio In LC and exported as .mp4
When I uploaded the .mp4 in YouTube. It said 'the video could not be processed.'
I now exported the same project from LC as .mkv, YouTube processed it and it got published with no errors.

**Comment by mifi** [comment]
closing due to inactivity
